go.mod

```diff
@@ -41,30 +40,30 @@ require (
 	github.com/hashicorp/hcl/v2 v2.9.1
 	github.com/hinshun/vt10x v0.0.0-20180616224451-1954e6464174
 	github.com/imdario/mergo v0.3.12
+	github.com/kubevela/prism v0.0.0-20220510071112-8a71d6d2d142
```

Comment:
we can make a fixed version for prism

Reply:
Later will release official versions, and then update this dependency.